python/docs/dlis/errors.rst

```diff
@@ -0,0 +1,179 @@
+Errors reference
+================
+
+If you received an error while working with dlis file, you might find
+explanation for it in this section.
+
+.. note::  This section doesn't describe *all* the errors you might see and not
+           *all* the reasons why these messages appear, but the most *common*
+           ones.
+
+.. note::  Please note that error messages might get updated while this sections
+           might not be, so treat them more as a hint than as a strict
+           guideline.
+
+.. contents:: :local:
+
+Errors on load
+--------------
+Load errors are in general non-recoverable and prevent user from further
+reading the file.
```

Comment:
1) Maybe start by mentioning that if load fails, it's most likely because the file is broken somehow. I don't think load will ever fail on a correctly written file? 

2) Mention errorhandler, as a potential remedy to the problem, and how it will read up-until the point of failure. Which makes it useful when the file is ill-formed at the end, but gives you no data when the file is ill-formed at the start 

3) A lot, if not all of the following errors are cause by the file being illformed in one of the following ways:
 - Trash at the start or end of the file
 -  Missing data in the middle
 - Truncated
 -  Zeroed out 

Which of the following errors that hits you, is dependent on exactly where in the file the above happens. I.e. the following errors are different manifestations of the same set of issues.

Reply:
1. maybe something like:
> Allthough there are still some features of the DLIS spec that dlisio doesn't implement full support for, it should be able to load all dlis files that conform to the spec. From experience we know that there exists a lot of files out there that doesn't strictly adhere to the spec, and these might case load to fail. It's rather common that files are truncated, the file is zeroed out from some point or that there are some extra "trash" bytes somewhere in the file. All of which can through dlisio parsing rutines off. 
2: 👍 
3: 👍